src/displayapp/screens/ApplicationList.cpp

```diff
@@ -60,7 +60,8 @@ std::unique_ptr<Screen> ApplicationList::CreateScreen2() {
            {Symbols::asterisk, Apps::Meter},
            {Symbols::paintbrush, Apps::Paint},
                   {Symbols::info, Apps::Notifications},
-                  {Symbols::paddle, Apps::Paddle},
+                  //TODO: Need to find the right place based on comments from JF
```

Comment:
You can take the slot of the 'Meter' app. This app is a simple demo app I added as a placeholder when there was no other apps available in InfiniTime. Now, we can remove it :)

That way, we'll keep the game Paddle in place ;)